docs/choreolib/auto-factory.md

```diff
@@ -1,109 +1,268 @@
 
-For a more advanced example of creating `AutoRoutine` see the [AutoRoutine](./auto-routines.md) documentation.
+To get started, create a new routine using `AutoFactory.newRoutine()`:
 
 ```java
-// Picking up where the last example left off
-public Robot extends TimedRobot {
-  ... //fields from previous example
-  /** A subsystem that controls the intake */
-  private final Intake intake = ...;
-  /** A subsystem that controls the shooter */
-  private final Shooter shooter = ...;
-  private final AutoChooser autoChooser;
-
-  public Robot() {
-    ... //code from previous example
-    autoChooser = new AutoChooser(autoFactory, "");
-    autoChooser.addRoutine("twoPieceAuto", this::twoPieceAuto);
-  }
-
-  // this would normally be in a separate file
-  private AutoRoutine twoPieceAuto(AutoFactory factory) {
-    final AutoRoutine routine = factory.newRoutine("twoPieceAuto");
-
-    final AutoTrajectory trajectory = routine.trajectory("twoPieceAuto");
-
-    routine.running()
-        .onTrue(
-            drive.resetOdometry(
-                    trajectory.getInitialPose()
-                        .orElseGet(
-                            () -> {
-                              routine.kill();
-                              return new Pose2d();
-                            }))
-                .andThen(trajectory.cmd())
-                .withName("twoPieceAuto entry point"));
-
-    trajectory.atTime("intake").onTrue(intake.extend());
-    trajectory.atTime("shoot").onTrue(shooter.launch());
+// Creates a new routine with the name "exampleRoutine"
+AutoRoutine routine = autoFactory.newRoutine("exampleRoutine");
+```
+
+The "entrance" to all routines is the `AutoRoutine.running()` trigger. You should bind the first command you want to execute to this trigger.
+
+```java
+routine.running().onTrue(Commands.print("Started the routine!"));
+```
+
+Traditional triggers can also be used in conjunction with an auto routine. However, they will experience "hygiene" issues if not used correctly, causing them to be active even when the routine is not. You must either:
+
+- Use `AutoRoutine.observe()` to ensure the trigger is only active when the routine is running
+- Conjoin a trigger to one created by `AutoRoutine` or `AutoTrajectory`
+
+```java
+Trigger myTrigger = new Trigger(() -> condition);
+
+// Safe
+routine.observe(myTrigger).onTrue(Commands.print("Foo"));
+routine.running().and(myTrigger).onTrue(Commands.print("Bar"));
+
+// Unsafe
+myTrigger.onTrue(Commands.print("Foo"));
+myTrigger.and(routine.running()).onTrue(Commands.print("Bar"));
+```
+
+Trajectories can be loaded using `AutoRoutine.trajectory()`, which will return an `AutoTrajectory` ([Java](/api/choreolib/java/choreo/auto/AutoTrajectory.html)). The `AutoTrajectory` class exposes multiple triggers for you to attach reactive logic to, as well as `AutoTrajectory.cmd()` for scheduling the trajectory.
+
+```java
+public AutoRoutine pickupAndScoreAuto() {
```

Comment:
Can you show some `atTime` examples as well?

Reply:
I meant to use `atTime()` originally in the example, I made the change and added an annotation for `atPose()`